### ISSUE TITLE
Could you offer specified commands for ALLEGROHAND and SHADOWHAND?

### ISSUE THREAD

**Issue by Charlie0257** [Improbable-AI/random-latent-exploration#1]
Hi @srinathm1359 
Thanks for your great work!

Unfortunately, I can't reproduce your results about **ALLEGROHAND and SHADOWHAND** mentioned in the paper.
Could you offer specified commands for ALLEGROHAND and SHADOWHAND?

Thanks for any help:)

Best,
Charlie

**Comment by srinathm1359** [comment]
Try adding the following flags: `--num_envs 8192 --int_coef 0.01 --norm_rle_features`. Hope this is helpful!

**Comment by Charlie0257** [comment]
Thanks for your reply! @srinathm1359 
I'm so sorry to reply late :(

I have the following questions I would like to ask about **Figure.7** of paper——
1. In the  **Figure.7**, I notice the label of the x-axis is **Progress**, does it mean the 600M Steps in the code?
2. I tried your suggested hyperparameters, but the performance seems to have some gap compared to your results in the **Figure.7**, Can you take a look at my return curve and complete hyperparameters？

```
anneal_lr:false
batch_size:65,536
capture_video:false
clip_coef:0.2
clip_vloss:false
cuda:true
ent_coef:0
env_id:"AllegroHand"
exp_name:"ppo_rle"
ext_coef:1
feature_size:32
gae_lambda:0.95
gamma:0.99
gpu_id:0
int_coef:0.01
int_gamma:0.99
int_vf_coef:0.5
learning_rate:0.001
max_grad_norm:0.5
minibatch_size:16,384
norm_adv:true
norm_rle_features:true
num_envs:8,192
num_iterations_feat_norm_init:1
num_minibatches:4
num_steps:8
record_video_step_frequency:1,464
reward_scaler:0.01
save_rle:false
seed:1
switch_steps:16
target_kl:null
tau:0.005
torch_deterministic:true
total_timesteps:600,000,000
track:true
update_epochs:5
vf_coef:2
```
![Screenshot from 2024-11-25 10-01-40](https://github.com/user-attachments/assets/a517bbef-930a-4dfc-b605-ac0aac498e07)
![Screenshot from 2024-11-25 10-01-46](https://github.com/user-attachments/assets/f4e98696-2dd2-4949-a486-17d0ffdd0b01)

My task is "AllegroHand", thanks for any advice!

Have a nice day!
Charlie

**Comment by srinathm1359** [comment]
Hey Charlie,
The metric we use in all the experiments is `charts/episode_return`. It looks like the score here is around 650, which is in the shaded area of the RLE performance curve in figure 7, so this checks out for allegrohand. Training more seeds should also give a better picture.

**Comment by Charlie0257** [comment]
Thanks for your patient reply!
